code/controllers/subsystems/trade.dm

```diff
@@ -112,13 +117,14 @@ SUBSYSTEM_DEF(trade)
 			if(station.uid == target_uid)
 				station.recommendations_needed -= 1
 				if(!station.recommendations_needed)
-					discovered_stations += station
+					discovered_stations |= station
+					//GLOB.entered_event.register(station.overmap_location, station, station/proc/discovered)
```

Comment:
```suggestion
					discovered_stations |= station
```
Are you leaving that comment for any particular reason? (a future PR you plan to do for instance)
If it was used for debug, no need to include it in the PR